examples/tsconfig_external/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_config", "ts_project")
+
+ts_config(
+    name = "config",
+    src = "tsconfig.json",
+    deps = [
+        "//examples:node_modules/@tsconfig/node16-strictest-esm"
+    ],
+)
+
+ts_project(
```

Comment:
Do we need a target to actually run this as a test?

Reply:
yes. we need to assert that the ts_project worker can read the external tsconfig properly. ts_config target by itself is not enough to assert that.

Reply:
I mean adding a `build_test` or something to create an actual test target...

Reply:
it's not needed.

Reply:
A lot of them do have test targets and I think we should move that direction. Why not?

Reply:
examples meant to be for users to understand how to do `x` and none of them has a `build_test` target. If we are going to start putting build_test to all of the examples that should be filed and fixed separately. I just follow the convention that we have today.

Reply:
About 2/3 seem to have tests already, mainly `build_test` and `assert_contains`.

I do think they should all have tests though, probably either `build_test`, `assert_contains` or `write_source_file`. Want to do that in a followup? Or maybe I will 🤷